### PR TITLE
Prep for release 1.1.0

### PR DESCRIPTION
**Pull request by IAlibay**
Sets versions for 1.1.0 ahead of testpypi upload

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Review by richardjgowers — APPROVED** [review]

File: package/CHANGELOG [hunk 1]
```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/??  richardjgowers, IAlibay, tylerjereddy, xiki-tempula, lilyminium,
+04/28/21  richardjgowers, IAlibay, tylerjereddy, xiki-tempula, lilyminium,
```
Comment (lilyminium) [inline package/CHANGELOG:16]: Do we really do MM/DD/YY?

**Review by lilyminium — APPROVED** [review]
LGTM although I have strong opinions about the date format

**Comment by lilyminium** [comment]
Could you merge when you're ready @IAlibay? This'll kick off a new "stable" set of docs at docs.mdanalysis.org since there's no `dev` anymore.

File: package/CHANGELOG (hunk 1 above)
Comment (IAlibay) [reply package/CHANGELOG:16]: Maybe we should switch over to ISO 8601? I'd be up for it if you want to propose it in a new issue 🙃

**Pull request merged**